docs/guide/static-deploy.md

```diff
@@ -187,6 +187,29 @@ The Netlify CLI will share with you a preview URL to inspect. When you are ready
 $ ntl deploy --prod
 ```
 
+## Cloudflare Pages
+
+1. Install [Wrangler CLI](https://developers.cloudflare.com/workers/wrangler/get-started/)
+2. Authenticate Wrangler with your Cloudflare account using `wrangler login`
+3. Run your build command 
+4. Deploy using `npx wrangler pages publish <Build-directory>`
```

Comment:
```suggestion
1. Install [Wrangler CLI](https://developers.cloudflare.com/workers/wrangler/get-started/).
2. Authenticate Wrangler with your Cloudflare account using `wrangler login`.
3. Run your build command.
4. Deploy using `npx wrangler pages publish dist`.
```

Added full-stop to be consistent with the other examples. And I think we should use `dist` instead of `<Build-directory>` too as it's what users have most of the time, and also other examples use it.